perf/smallbank_workload/src/main.rs

```diff
@@ -147,7 +147,7 @@ fn create_load_subcommand_args<'a, 'b>() -> App<'a, 'b> {
 
 fn run_load_command(args: &ArgMatches) -> Result<(), Box<Error>> {
     let max_txns: usize = match args.value_of("max-batch-size")
-        .unwrap_or("100")
+        .unwrap_or("1")
         .parse() {
             Ok(n) => n,
             Err(_) => 0,
```

Comment:
This error should be returned, not silently set to `0`

Reply:
Yeah, I will update that.

Reply:
Thinking about it, the next check is whether max_txns is not equal to 0. All failures to parse as unsigned int state that "max-batch-size must be a number greater than 0" which seems like a better statement to the user than panicking on a ParseIntError kind: INvalidDigit.